src/staticList.js

```diff
@@ -0,0 +1,190 @@
+import fetch from "node-fetch";
+import fs from "fs";
+import path from "path";
+import _ from "lodash";
+
+const cowListUrl = "https://files.cow.fi/tokens/CowSwap.json";
+const coinGeckoListUrl = "https://files.cow.fi/tokens/CoinGecko.json";
+const coinGeckoIdListUrl = "https://api.coingecko.com/api/v3/coins/list";
+
+const LIST_DIR = path.join("src", "lists");
+const IDS_FILE_NAME = "id-list.json";
+const IDS_FILE_PATH = path.join(LIST_DIR, IDS_FILE_NAME);
+
+const STATIC_LIST_NAME = "static-list.json";
+const STATIC_LIST_PATH = path.join(LIST_DIR, STATIC_LIST_NAME);
+
+const STATIC_LIST_PARSED_NAME = "static-list-parsed.json";
+const STATIC_LIST_PARSED_PATH = path.join(LIST_DIR, STATIC_LIST_NAME);
+
+const CUSTOM_DESCRIPTION_PATH = path.join("src", "files", "description.json");
+
+async function fetchFromCoingecko(url) {
+  console.log(`Fetching ${url}`);
+  const res = await fetch(url);
+
+  if (!res.ok) {
+    throw new Error(`Error fetching list ${url}`, res);
+  }
+
+  const json = await res.json();
+  return json;
+}
+
+async function writeFile(dir, filename, input) {
+  const data = JSON.stringify(input, null, 4);
+  const filePath = path.join(dir, filename);
+
+  console.log(`Writing file ${filePath}`);
+
+  if (!fs.existsSync(dir)) {
+    fs.mkdirSync(dir, { recursive: true });
+  }
+
+  fs.writeFileSync(filePath, data);
+}
+
+function sleep(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+async function getTokenLists() {
+  // Get token lists
+  const cowList = await fetchFromCoingecko(cowListUrl);
+  const coinGeckoList = await fetchFromCoingecko(coinGeckoListUrl);
+
+  // Combine 2 lists
+  const combined = [...cowList.tokens, ...coinGeckoList.tokens];
+
+  console.log("Combined tokens", combined.length);
+
+  return combined;
+}
+
+async function getIds() {
+  // If ID's file exist
+  if (fs.existsSync(IDS_FILE_PATH)) {
+    const json = fs.readFileSync(IDS_FILE_PATH, "utf8");
+    return JSON.parse(json);
+  }
+
+  const combined = await getTokenLists();
+
+  // Get Coingecko id list
+  const coinGeckoIdList = await fetchFromCoingecko(coinGeckoIdListUrl);
+
+  // Crate unique list by symbols from those 2
+  const uniq = _.uniqBy(combined, ({ symbol }) => symbol);
```

Comment:
Do we need lodash for this?

Other than that, symbol is not really unique, shouldn't we filter by address instead?

Reply:
For example, there's this case in the added `id-list` file:

![image](https://github.com/cowprotocol/token-lists/assets/43217/c6ebb936-8ded-4d49-845f-f03b507285e5)